docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,49 +1,54 @@
 - Are there plans for follow up PR’s to further fix this bug or develop this feature?
 
-
-# Review Checklist
+# Checklist
+_PR Creator - Please cater the checklist to fit the review needed for your code changes._
+_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._
 
 ## General
-
-- [ ] Are the tests passing locally and on Travis?
-- [ ] Is the documentation up to date?
-
-## Components
-- [ ] Has a storybook story been created or updated?
-- [ ] Is the component accessible? 
-  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
-  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
-  - [ ] Is it passing accessibility checks in the storybook?
-
-## Apps
-
-- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
-- [ ] Does it work on mobile?
-- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?
-
-## Publishing
-
-- [ ] Is the changelog updated?
-- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?
-
-## Post-merging
-
-- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
-- [ ] Is the new feature working or bug now fixed?
-  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
-- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
-  - [ ] Is this approved by our designer?
-- [ ] Is this ready for production deployment?
+- [ ] Tests are passing locally and on Github
+- [ ] Documentation is up to date and changelog has been updated if appropriate
+- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
+- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
+- [ ] FEM works in a mobile browser
+
+## General UX
+Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
+- [ ] All pages of a FEM project load: Home Page, Classify Page, and About Pages
+- [ ] Can submit a classification
+- [ ] Can sign-in and sign-out
+- [ ] The component is accessible
+  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
+  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
+  - It is passing accessibility checks in the storybook
+
+
+## Bug Fix
+- [ ] The PR creator has listed user actions to use when testing if bug is fixed
+- [ ] The bug is fixed
+- [ ] Unit tests are added or updated
```

Comment:
I’m not sure of the exact wording I’d use for this but: is there a test that catches the bug?

This isn’t true for all bugs, but a fix that starts with a failing test, followed by changes that pass the test, is easier to review.

Reply:
Good idea! I think the refactoring of unit tests changes a lot on a case by case basis, and TDD as a concept is a better fit for our frontend team playbook. I'll leave the wording as it for now, and we can re-edit in the future.